### PR TITLE
Bug 2037628: added test-id to kms flows

### PR DESCRIPTION
**Pull request by RishabhKodes**
Added the test id in the kms flow for automation

### CODE REVIEW

**Comment by RishabhKodes** [comment]
> @RishabhKodes: This pull request references [Bugzilla bug 2037628](https://bugzilla.redhat.com/show_bug.cgi?id=2037628), which is invalid:
> 
> * expected the bug to target the "4.11.0" release, but it targets "---" instead
> 
> Comment `/bugzilla refresh` to re-evaluate validity if changes to the Bugzilla bug are made, or edit the title of this pull request to link to a different bug.

/bugzilla refresh

**Review by afreen23 — CHANGES_REQUESTED** [review]
Add test ids to all FileUploads

/ok-to-test

File: frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-vault-modal.tsx [hunk 1]
```diff
@@ -253,6 +254,7 @@ export const AdvancedVaultModal = withHandlePromise((props: AdvancedKMSModalProp
                 maxSize: KMSMaxFileUploadSize,
                 onDropRejected: () => setError(KMSFileSizeErrorMsg),
               }}
+              data-test="kms-service-cert"
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-vault-modal.tsx:257]:
```suggestion
              data-test="kms-service-client-cert"
```

File: frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-vault-modal.tsx [hunk 2]
```diff
@@ -273,6 +275,7 @@ export const AdvancedVaultModal = withHandlePromise((props: AdvancedKMSModalProp
                 maxSize: KMSMaxFileUploadSize,
                 onDropRejected: () => setError(KMSFileSizeErrorMsg),
               }}
+              data-test="kms-service-key"
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-vault-modal.tsx:278]:
```suggestion
              data-test="kms-service-client-private-key"
```

**Review by afreen23 — CHANGES_REQUESTED** [review]

**Review by afreen23 — APPROVED** [review]
/lgtm

**Pull request merged**